src/org/sosy_lab/java_smt/test/InterpolatingProverTest.java

```diff
@@ -234,6 +234,7 @@ private void requireTreeItp() {
     assume()
         .withMessage("Solver does not support tree-interpolation.")
         .that(solver)
+        // .isAnyOf(Solvers.Z3, Solvers.SMTINTERPOL, Solvers.PRINCESS);
```

Comment:
Why this change?

Reply:
I wanted to test out `Tree Interpolation` with CVC5. Forgot to change it back.